Find a functionally equivalent code:android.media.MediaFocusControl.setNewRcClientOnDisplays_syncRcsCurrc:COMMENT
Method Modifier: private     
Comment:/**
 * Update the remote control displays with the new "focused" client generation
 */

@@ -1,5 +1,5 @@
 {
-    synchronized (mRCStack) {
+    synchronized (mPRStack) {
         if (mRcDisplays.size() > 0) {
             final Iterator<DisplayInfoForServer> displayIterator = mRcDisplays.iterator();
             while (displayIterator.hasNext()) {

